rpc/src/v1/types/action.rs

```diff
@@ -55,36 +53,67 @@ pub enum Action {
     Custom(Bytes),
 }
 
-impl Action {
+#[derive(Debug, Deserialize, Serialize)]
```

Comment:
You don't need to derive `Deserizlie` for it.

Reply:
Fixed.